service/rtc/session.go

```diff
@@ -62,11 +62,19 @@ type session struct {
 	// Sender (publishing side)
 	outVoiceTrack        *webrtc.TrackLocalStaticRTP
 	outVoiceTrackEnabled bool
-	screenStreamID       string
-	outScreenTracks      map[string][]*webrtc.TrackLocalStaticRTP
-	outScreenAudioTrack  *webrtc.TrackLocalStaticRTP
-	remoteScreenTracks   map[string]*webrtc.TrackRemote
-	screenRateMonitors   map[string]*RateMonitor
+	remoteVoiceTrack     *webrtc.TrackRemote
+	// Screen sharing
+	screenStreamID         string
+	outScreenTracks        map[string][]*webrtc.TrackLocalStaticRTP
+	outScreenAudioTrack    *webrtc.TrackLocalStaticRTP
+	remoteScreenAudioTrack *webrtc.TrackRemote
+	remoteScreenTracks     map[string]*webrtc.TrackRemote
+	screenRateMonitors     map[string]*RateMonitor
+	// Video
+	videoStreamID     string
+	outVideoTracks    map[string][]*webrtc.TrackLocalStaticRTP
+	remoteVideoTracks map[string]*webrtc.TrackRemote
+	videoRateMonitors map[string]*RateMonitor
```

Comment:
I'm also wondering about duplication here (screen vs video tracks). Maybe it's unavoidable?